### DISCUSSION TITLE
Guitar Tuner for X & Y Belt Tension. What frequency do you use for Ender 3 V2?

### DISCUSSION THREAD

**Discussion by BlueWings172** [Jyers/Marlin#1708] [General Discussion]
Hi

I've been trying to figure out why i'm getting such terrible prints lately and stumbled on posts talking about belt tension and using a guitar tuning app to determine if the belts of a 3D printer are properly tensioned.

I may have over tensioned my Ender 3 V2 belts and have always wondered what it the right level of tension for the belts and what would happen when the distance between the belt teeth increases due to stretching over time.

Have anyone tried using tuners ? and if yes, what are the frequencies I should be looking for for X and Y belts?

thanks

**Comment by Cal-LF** [comment]
No tuner here. Tension to snug. The threaded screw should be just below flush.
